### PR TITLE
chore: advertise shorter openapi operation ids

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

shorter operationIds for the main service (determiend) would result in friendlier names for generated code
while not adding collision risk.

## Checklist

- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ioga — APPROVED** [review]
makes sense

File: webui/react/src/components/TrialLogPreview.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const TrialLogPreview: React.FC<PropsWithChildren<Props>> = ({
 
   const fetchTrialLogs = useCallback((trialId: number, time: string, canceler: AbortController) => {
     consumeStream(
-      detApi.StreamingExperiments.determinedTrialLogs(
+      detApi.StreamingExperiments.trialLogs(
```
Comment (hkang1) [inline webui/react/src/components/TrialLogPreview.tsx:39]: this is sooooo much nicer!!! :)

**Review by hkang1 — APPROVED** [review]
👏           👏        👏       👏     👏   👏 👏 👏 👏 👏 🥇 🥳  (slow clap :))
Love this PR!

**Comment by hamidzr** [comment]
thanks for the review! I'll rebase just to make sure there aren't any new references added in the past 2 days

**Comment by hamidzr** [comment]
there are four Enums that are not named using the same format (not an operation) that I'm not updating here
those are a bit more involved. We might do that in another pr.

```
* defintion determinedcheckpointv1State
* defintion determinedcontainerv1State
* defintion determinedexperimentv1State
* defintion determinedtaskv1State
```

**Pull request merged**